### PR TITLE
Recovery Cert Change - meaningful information (EXPOSUREAPP-12063)

### PR DESCRIPTION
**Pull request by BMItr**
This PR addresses Exposureapp-[12063](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-12063).
Contains more "meaningful informations" for the recovery certificates (e.g. date of the first positive test).

How to test:
- scan a recovery certificate (cli)
- check the changes ui component  (card is displayed together with "positive test..")
- affected cards: Dcc PersonView (Overview), Recycle Bin, Reissuance Consent View, Dcc Validation Service

Screenshot:
image

### CODE REVIEW

**Review by KathrinLuecke — APPROVED** [review]
UA approved

File: Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/covidcertificate/person/ui/details/PersonDetailsFragmentTest.kt [hunk 1]
```diff
@@ -408,7 +408,7 @@ class PersonDetailsFragmentTest : BaseUITest() {
             every { fullName } returns "Andrea Schneider"
             every { uniqueCertificateIdentifier } returns "RN:UVCI:01:AT:858CC18CFCF5965EF82F60E493349AA5#K"
             every { dateOfBirthFormatted } returns "1981-03-20"
-            every { validUntil } returns Instant.parse("2021-03-31T11:35:00.000Z").toLocalDateUserTz()
+            every { testedPositiveOn } returns Instant.parse("2021-05-23T11:35:00.000Z").toLocalDateUserTz()
```
Comment (LukasLechnerDev) [inline Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/covidcertificate/person/ui/details/PersonDetailsFragmentTest.kt:411]: `23.05.` is my birth day 😎

**Review by LukasLechnerDev — APPROVED** [review]
LGTM!

**Review by mtwalli — APPROVED** [review]

File: Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/covidcertificate/person/ui/details/PersonDetailsFragmentTest.kt (hunk 1 above)
Comment (BMItr) [reply Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/covidcertificate/person/ui/details/PersonDetailsFragmentTest.kt:411]: @LukasLechnerDev  thats why it put it in ;-)

**Pull request merged**